lib/data_manager.rb

```diff
@@ -0,0 +1,80 @@
+require 'json'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+
+class DataManager
+  def initialize(people, books, rentals)
+    @people = people
+    @books = books
+    @rentals = rentals
+  end
+
+  def save_data
+    people_data, books_data, rentals_data = serialize_data
+
+    File.write('./lib/data/people.json', JSON.generate(people_data))
+    File.write('./lib/data/books.json', JSON.generate(books_data))
+    File.write('./lib/data/rentals.json', JSON.generate(rentals_data))
+  end
```

Comment:
[OPTIONAL] When I exited the application, I got the following error: 
![image](https://github.com/mwismann/oop_school_library/assets/93520178/820706c5-8bda-40d3-a699-68692ea9dddf)

The files themselves don't need to exist for the code to work, but the directory `data` needs to exist.This directory probably doesn't exist because Git ignores empty directories when pushing changes to a remote repo. In any case, maybe you should add the folder `data` with the three `json` files to it, each `json` file having an empty array to avoid confusion for other users running your code in the future.